examples/semi_analytical_sedimentation.jl

```diff
@@ -0,0 +1,83 @@
+"Sedimentation of an initial layer of liquid with a single-mode gamma dist"
+
+using LinearAlgebra
+using Plots
+using Interpolations
+
+using Cloudy.KernelTensors
+using Cloudy.ParticleDistributions
+using Cloudy.Sources
+
+import Cloudy.ParticleDistributions: density, nparams, density_func
+
+function initial_condition(z)
```

Comment:
Maybe create a separate "utils/rainshaft_helpers.jl" file and put the initial condition, sedimentation functions, and analytical solution details in there. Then call it in a single rainshaft example that includes both the analytical and empirical